src/js/views/admin/sidebar/clinician/clinician-sidebar_views.js

```diff
@@ -91,79 +140,160 @@ const SidebarView = View.extend({
+      },
+    ]);
+
+    const optionlist = new Optionlist({
+      ui: this.ui.menu,
+      uiView: this,
+      headingText: i18n.sidebarView.menuOptions.headingText,
+      itemTemplate: hbs`<span class="sidebar__delete-icon">{{far "trash-alt"}}</span>{{ @intl.admin.sidebar.clinician.clinicianSidebarViews.sidebarView.menuOptions.delete }}`,
+      lists: [{ collection: menuOptions }],
+      align: 'right',
+      popWidth: 248,
+    });
+
+    optionlist.show();
+  },
+  cloneClinician() {
+    // NOTE: creates a new clone from the truth for cancelable editing
+    if (this.clonedClinician) this.stopListening(this.clonedClinician);
+    this.clonedClinician = this.clinician.clone();
   },
-  showEmail() {
-    this.showChildView('email', new EmailView({ model: this.model }));
+  showName(error) {
+    this.showChildView('name', new NameView({
+      model: this.clonedClinician,
+      error,
+    }));
+  },
+  showEmail(error) {
+    this.showChildView('email', new EmailView({
+      model: this.clonedClinician,
+      error,
+    }));
   },
   showAccess() {
-    const isDisabled = this.model.isNew();
-    const accessComponent = new AccessComponent({ model: this.model, state: { isDisabled } });
+    const isDisabled = this.clinician.isNew();
+    const accessComponent = new AccessComponent({ model: this.clinician, state: { isDisabled } });
 
     this.listenTo(accessComponent, 'change:access', accessType => {
-      this.model.save({ access: accessType });
+      this.clinician.save({ access: accessType });
     });
 
     this.showChildView('access', accessComponent);
   },
   showRole() {
-    const isDisabled = this.model.isNew();
-    const roleComponent = new RoleComponent({ model: this.model, state: { isDisabled } });
+    const isDisabled = this.clinician.isNew();
+    const roleComponent = new RoleComponent({ model: this.clinician, state: { isDisabled } });
 
     this.listenTo(roleComponent, 'change:role', role => {
-      this.model.saveRole(role);
+      this.clinician.saveRole(role);
     });
 
     this.showChildView('role', roleComponent);
   },
   showGroups() {
-    this.showChildView('groups', new GroupsComponent({
-      clinician: this.model,
-    }));
+    this.showChildView('groups', new GroupsComponent({ clinician: this.clinician }));
   },
   showSave() {
-    if (!this.model.isValid()) {
+    if (!this.clonedClinician.isValid() || !_.isEmpty(this.state.get('errors'))) {
       this.showDisabledSave();
       return;
     }
 
-    this.showChildView('save', new SaveView({ model: this.model }));
+    this.showChildView('save', new SaveView({ model: this.clonedClinician }));
   },
   showDisabledSave() {
     this.showChildView('save', new DisabledSaveView());
   },
   showForm() {
-    if (this.model.isNew()) {
-      this.listenTo(this.model, 'change:name change:email', this.showSave);
+    if (_.isEmpty(this.state.get('errors'))) {
+      this.cloneClinician();
+    }
+
+    if (this.clinician.isEditable()) {
```

Comment:
I think it'll be ok if we leave the error state up until the following submission